common/djangoapps/mailchimp_pipeline/tests/test_tasks.py

```diff
@@ -1,39 +1,40 @@
+"""
```

Comment:
Since we are deleting tab spaces in docstrings from the above files, it would be great to do the same for docstrings in this file too, in addition, following the same format where ever arguments and return type is to be mentioned would also help.